todo/services/task_service.py

```diff
@@ -234,3 +233,9 @@ def create_task(cls, dto: CreateTaskDTO) -> CreateTaskResponse:
                     ],
                 )
             )
+
+    @classmethod
+    def delete_task(cls, task_id: str) -> None:
+        deleted_task_model = TaskRepository.delete_by_id(task_id)
+        if deleted_task_model is None:
+            raise TaskNotFoundException(task_id)
```

Comment:
```suggestion
    def delete_task(cls, task_id: str) -> None:
        deleted_task_model = TaskRepository.delete_by_id(task_id)
        if deleted_task_model is None:
            raise TaskNotFoundException(task_id)
        return None
```

should we return explicitly?